openstack/cronus/templates/rhea/_config.yaml.tpl

```diff
@@ -19,6 +19,23 @@ rhea:
     endpointType: {{ .Values.config.endpointType }}
 {{- end }}
   auth:
+    keystone:
+{{- if .Values.config.keystone }}
+{{- range $key, $value := .Values.config.keystone }}
```

Comment:
FYI, not that you have to change it, but it would should also be possible to do:
```
# values.yaml
keystone: |
  myKey1: "test"
  myKeys2: "test2"
 ---
 # config.yaml.tpl
     keystone:
 {{.Values.keystone  | indent 4 }}
 ```
 (maybe indent 6)

Reply:
It would be nice if there was a better way so as not to use a string literal '|', but ATM I am not seeing another way of just copying a map to the template. Anyway, do which ever you think is best. Just wanted to comment this alternate way :)

Reply:
Ah yes, I am theoretically aware of it. :-) I find it most useful for blobs.